instances/latest/terminologies/disease/macularDegeneration.jsonld

```diff
@@ -0,0 +1,21 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.om-i.org/props/"
+  },
+  "@id": "https://openminds.om-i.org/instances/disease/macularDegeneration",
+  "@type": "https://openminds.om-i.org/types/Disease",
+  "definition": "Degenerative changes in the macula lutea of the retina.",
+  "description": "Deterioration of the macula lutea in the retina; may be inherited, drug induced, or due to aging; leads to a severe loss of central vision while peripheral vision is retained.",
+  "interlexIdentifier": "http://uri.interlex.org/base/ilx_0106448",
+  "knowledgeSpaceLink": null,
```

Comment:
found it there: https://knowledge-space.org/wiki/BIRNLEX:12811#macular-degeneration-1 (it's the BIRNLEX entry, but also states the DOID under 'defined by')